lib/DDG/Meta/Data.pm

```diff
@@ -122,6 +125,26 @@ sub get_ia {
     return $m;
 }
 
+sub _satisfy {
+	my ($ia, $by, $lookup) = @_;
+	return $by->($ia, $lookup) if ref $by eq 'CODE';
+	my $pby = $ia->{$by};
+	ref $lookup eq 'CODE' ? $lookup->($pby) : $pby eq $lookup;
+}
+
+# get_ias(repo => 'goodies', dev_milestone => 'live'...)
+# Lookups combine as an AND operation.
+sub get_ias {
+	my (@lookups) = @_;
+	my @results = @ia_container;
+	foreach (pairs @lookups) {
```

Comment:
Okay, so just support things like `id => [qw(a b c)]` being `id` is either `a`, `b`, or `c`?

With multiple lookups being specified meaning they should all be satisfied?